src/covvfit/plotting/_timeseries.py

```diff
@@ -2,10 +2,15 @@
 
 import matplotlib.lines as mlines
 import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+from jaxtyping import Array, Float
 
-colors_covsp = {
+Variant = str
```

Comment:
This annotations is just for total beginner virologist like myself, so I don't confuse variants with color names.